src/ray/gcs/gcs_server/test/gcs_health_check_manager_test.cc

```diff
@@ -71,6 +71,7 @@ class GcsHealthCheckManagerTest : public ::testing::Test {
     auto channel = grpc::CreateChannel("localhost:" + std::to_string(port),
                                        grpc::InsecureChannelCredentials());
     server->Run();
+    server->GetServer().GetHealthCheckService()->SetServingStatus(node_id.Hex(), true);
```

Comment:
@edoakes  this is the place.

So basically, if this line is not there, this test will fail because health check manager will check the specific service name.

Reply:
Yeah I saw it